### PR TITLE
Fix multisearch query struct

### PR DESCRIPTION
**Pull request by rahulPalliyalil**
### What

When trying to run the dp-search-api GET search api against elastic version 7.10, it is not getting the documents and resulting in 500 internal server error with a message as failed to perform multi search. 

So this pr is to basically fix the issue by restructuring the template according to the search object with Header and Query attributes. 

### How to review

This is done according to the trello ticket: https://trello.com/c/ahFHjF0R/1239-bug-template-use-me-for-new-bugs. 

So would be good to compare both and see if this makes sense. 

### Who can review

Anyone except me.

### CODE REVIEW

File: query/search.go [hunk 1]
```diff
@@ -88,15 +88,15 @@ func SetupV710Search() (*template.Template, error) {
...
+func (sb *Builder) BuildSearchQuery(ctx context.Context, q, contentTypes, sort string, topics []string, limit, offset int, esVersion710 bool) ([]byte, error) {
 	reqParams := searchRequest{
 		Term:  q,
 		From:  offset,
 		Size:  limit,
 		Types: strings.Split(contentTypes, ","),
 		//Topic:            topics, // Todo: This needs to be reintroduced when migrating to ES 7.10
 		SortBy:           sort,
-		AggregationField: "_type",
+		AggregationField: "type",
```
Comment (nshumoogum) [inline query/search.go:99]: By changing this does it not result in an issue with the API interacting with ES 2.2?

File: query/search_test.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func TestBuildSearchQuery(t *testing.T) {
 		So(queryString, ShouldContainSubstring, "Size=2")
 		So(queryString, ShouldContainSubstring, "Types=[ta tb]")
 		So(queryString, ShouldContainSubstring, "SortBy=relevance")
-		So(queryString, ShouldContainSubstring, "AggregationField=_type")
+		So(queryString, ShouldContainSubstring, "AggregationField=type")
```
Comment (nshumoogum) [inline query/search_test.go:41]: This should have never changed

**Review by nshumoogum — CHANGES_REQUESTED** [review]
I turned on ES710 flag in component tests and this results in several test failures; we should extend the component tests for ES710.

So run the same features with two separate configurations ES710 flag switched on (true) and the other where this flag is switched off (false)

**Comment by rahulPalliyalil** [comment]
> I turned on ES710 flag in component tests and this results in several test failures; we should extend the component tests for ES710.
> 
> So run the same features with two separate configurations ES710 flag switched on (true) and the other where this flag is switched off (false)

As mentioned during standup, will activate the component tests after fixing the transform completely, which will be the next pr. I will extend the current ticket to include the transform fix if its alright ?

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**